code/modules/antagonists/cult/runes.dm

```diff
@@ -57,11 +59,21 @@ Runes can either be invoked by one's self or with many different cultists. Each
 
 /obj/effect/rune/attackby(obj/I, mob/user, params)
 	if(istype(I, /obj/item/melee/cultblade/dagger) && iscultist(user))
+		if(log_when_erased)
+			var/confirm = alert(user, "Erasing this [src.cultist_name] rune might be against your goal to summon Nar'Sie.", "Begin to erase the [src.cultist_name] rune?", "Proceed", "Abort")
```

Comment:
This and everywhere else, change `src.cultist_name` to `cultist_name`--src is implied.